interface/orders/orders_results.php

```diff
@@ -626,7 +626,7 @@ function validate(f) {
                 "form_report_status[$lino]",
                 'proc_rep_status',
                 $report_status,
-                xl('Report Status'),
+                xlt('Report Status'),
```

Comment:
Note that should only by htmlescaping stuff right before it is sent to html output. Should not escape above since this is a function call (which will do it's own html escaping on the html output). So should revert above line.

Reply:
That makes a lot of sense, thank you for the explanation!